x-pack/plugins/cases/public/components/case_view/does_not_exist.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiEmptyPrompt, EuiButton } from '@elastic/eui';
+import * as i18n from './translations';
+import { CasesNavigation } from '../links';
+interface Props {
+  allCasesNavigation: CasesNavigation;
+  caseId: string;
+}
```

Comment:
I know this comment is very picky 😄 but it is difficult to navigate without spaces between functions/components/variables. Just wanted to mentioned it. No need to change your style.